app/views/browse/show.html.erb

```diff
@@ -40,18 +40,30 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
```

Comment:
I think these two lines can be removed as they have been duplicated lines 43 and 44 and I don't think these classes need to be added twice?